jscomp/core/js_dump_import_export.ml

```diff
@@ -70,7 +70,11 @@ let exports cxt f (idents : Ident.t list) =
 
 (** Print module in ES6 format, it is ES6, trailing comma is valid ES6 code *)
 let es6_export cxt f (idents : Ident.t list) =
-  let outer_cxt, reversed_list =
+  P.at_least_two_lines f;
```

Comment:
I don't really like to add a new line when there is no other output here, but that's the best way I could come up with to keep the `/*  Not a pure module */` comment display in a good way.